go/vt/orchestrator/inst/analysis.go

```diff
@@ -122,6 +122,7 @@ type ReplicationAnalysis struct {
 	AnalyzedInstanceDataCenter                string
 	AnalyzedInstanceRegion                    string
 	AnalyzedKeyspace                          string
+	AnalyzedShard                             string
```

Comment:
We need the name of the shard that was analyzed too now that we want to restrict the number of tablets we want to refresh